cmd/mdatagen/templates/component_test.go.tmpl

```diff
@@ -1,5 +1,10 @@
 // Code generated by mdatagen. DO NOT EDIT.
 
+{{ if not supportsWindows }}
+//go:build !windows
+// +build !windows
```

Comment:
All the `+build` directives are removed from the project. No need to render this anymore